src/Runtime/LambdaRuntime.php

```diff
@@ -206,13 +206,19 @@ private function signalFailure(string $invocationId, \Throwable $error): void
             $errorMessage = $error->getMessage();
         }
 
+        $errorMessage = sprintf(
+            'Fatal error: %s in %s:%d',
+            $errorMessage,
+            $error->getFile(),
+            $error->getLine()
+        );
+
         // Log the exception in CloudWatch
         printf(
-            "Fatal error: %s in %s:%d\nStack trace:\n%s",
+            '{"errorType": "%s", "errorMessage": "%s", "stack": %s}',
+            get_class($error),
             $errorMessage,
-            $error->getFile(),
-            $error->getLine(),
-            $error->getTraceAsString()
```

Comment:
Thanks that definitely a good improvement.

We are loosing the file + line number in the error message. Do you know if notices/warnings/errors (i.e. everything except exceptions) include this information in the message? Else I think we should re-add those in the `else {` at the top of this function.

Reply:
Why do you think we're losing that? I kept the message exactly the same so that nothing would be actually lost/changed per se. The only true change is the format of the message. The `$error->getFile()` and `$error->getLine()` just moved a bit up

Reply:
I don't understand my comment either 🤦‍♂ 

Sorry ^^